tests/helpers/providerStateHelper.js

```diff
@@ -33,7 +33,7 @@ const givenGroupExists = (provider, group) => {
  */
 const givenFileExists = (provider, username, password, resource) => {
   return provider
-    .given('file exists', { username, password, fileName: encodeURIComponent(resource) })
+    .given('file exists', { username, password, fileName: encodeURIComponent(resource).split('%2F').join('/') })
```

Comment:
Why is this required here?, if it doesn't likes encoded filename then we can use `decodeURIComponent`

Reply:
It needs to encode files but there can be a path like `folder/file.txt`. In that case, it will also encode `/` which we don't want happen.